applications/luci-app-wireguard/Makefile

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2016-2017 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=WireGuard Status
+LUCI_DEPENDS:=+wireguard
```

Comment:
You might add the typical LUCI_PKGARCH line that tell that the package can be installed in all target architectures. Luci packages usually have no binaries, so there is no hardware target restriction. 

Most luci apps define
` LUCI_PKGARCH:=all `